ycms/cms/templates/patient/patient_card.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n %}
+<div class="bg-white rounded-lg shadow-lg p-4 w-full">
+    <!-- Patient Information Header -->
+    <div class="text-center mb-6">
+        <h2 class="text-2xl font-semibold">{% translate "Patient Information" %}</h2>
+    </div>
+    <!-- Patient Details -->
+    <div class="grid grid-cols-1 md:grid-cols-2 gap-4">
+        <!-- Personal Information -->
+        <div class="mb-2">
+            <h3 class="text-lg font-semibold mb-2">{% translate "Personal Details" %}</h3>
+            <p>
+                <span class="font-semibold">{% translate "Name" %}:</span> {{ patient.first_name }} {{ patient.last_name }}
+            </p>
+            <p>
+                <span class="font-semibold">{% translate "Age" %}:</span> {{ patient.age }}
+            </p>
+            <p>
+                <span class="font-semibold">{% translate "Gender" %}:</span> {{ patient.gender | title }}
+            </p>
+            <p>
+                <span class="font-semibold">{% translate "Accompanied" %}:</span>
+                {% if bed_assignment.accompanied %}
+                    <i icon-name="check" class="w-4 h-4 text-green-700"></i>
+                {% else %}
+                    <i icon-name="x" class="w-4 h-4 text-red-700"></i>
+                {% endif %}
+                <p>
+                    <span class="font-semibold">{% translate "Privately insured" %}:</span>
+                    {% if patient.insurance_type %}
+                        <i icon-name="check" class="w-4 h-4 text-green-700"></i>
+                    {% else %}
+                        <i icon-name="x" class="w-4 h-4 text-red-700"></i>
+                    {% endif %}
+                </p>
+            </p>
+        </div>
+        <!-- Medical Information -->
+        <div class="mb-2">
+            <h3 class="text-lg font-semibold mb-2">{% translate "Medical Details" %}</h3>
+            <p>
+                <span class="font-semibold">{% translate "Diagnosis code" %}:</span>
+                {{ bed_assignment.medical_record.diagnosis_code }}
+            </p>
+            <p>
+                <span class="font-semibold">{% translate "Note" %}:</span> {{ bed_assignment.medical_record.note }}
+            </p>
+            <p>
+                <span class="font-semibold">{% translate "Admission date" %}:</span>
+                {% if patient.current_admission_date %}{{ patient.current_admission_date }}{% endif %}
```

Comment:
```suggestion
                {{ patient.current_admission_date }}
```
`None` values (or nonexistent attributes, for that matter) will simply not be rendered, ther is no advantage to checking like this.

There are a bunch of these, please change them all, I just picked this by chance 😄

Reply:
It actually be rendered to string None which then shows on the page, it's not intuitive to see such word. 
![image](https://github.com/charludo/ycms/assets/18380064/3b101d9e-6231-403f-8293-2a0b127456bd)

Reply:
That is very strange, same thing renders as empty string for me (and AFAIK it should generally not show None, unless somewhere we are returning the string "None").

I guess leave it for now, I'll try to investigate.

Reply:
Ahhhhh, I get it now (I think, lol). `current_room`, `current_admission_date` and so on explicitly return None. Apparently, None is rendered as "None" (did not know that!).

On the other hand, things like `current_stay` will render as the empty string when the queryset is empty - I assume there's just some logic for that in the queryset's `__str__`.

I see three options:
- leave as is: works, but is a bit verbose IMO.
- use `{{ patient.current_admission_date|default_if_none="' }}` <- I would prefer this if you prefer to solve this in the template
- change the cached properties to return `""` instead of `None`; I don't think this should cause any troubles, since we are not testing for actual `None`-ness anywhere, only for falsy-ness. I would prefer this option, personally, as it is "set and forget", if we require those values somewhere else in a template it will just work